src/Storage/Storage.Management/StorageAccount/StorageAccountBaseCmdlet.cs

```diff
@@ -67,6 +67,22 @@ protected struct AccountAccessTier
             internal const string Hot = "Hot";
             internal const string Cool = "Cool";
         }
+        protected struct AzureBlobType
+        {
+            internal const string BlockBlob = "blockBlob";
+            internal const string PageBlob = "pageBlob";
+            internal const string AppendBlob = "appendBlob";
+        }
+        protected struct ManagementPolicyAction
+        {
+            internal const string TierToCool = "TierToCool";
+            internal const string TierToArchive = "TierToArchive";
+            internal const string Delete = "Delete";
+        }
+        protected struct ManagementPolicyVersion
```

Comment:
Where is this used?

Reply:
This is in the policy defination original, but removed now.
I will remove it from PSH code.